py2puml/inspection/inspectclass.py

```diff
@@ -119,34 +78,42 @@ def parse_class_type(
                     attr_type = f"{composition_rel}[{', '.join(component_names)}]"
                 else:
                     attr_type = attr_raw_type
-            uml_attr = UmlAttribute(attr_name, attr_type)
+            uml_attr = UmlAttribute(attr_name, attr_type, True)
```

Comment:
```suggestion
            uml_attr = UmlAttribute(attr_name, attr_type, static=True)
```

Reply:
agreed, it makes the source code more explicit. I changed that in other places too.